src/main/java/org/opentripplanner/model/Pathway.java

```diff
@@ -57,16 +71,45 @@ public void setTraversalTime(int traversalTime) {
         this.traversalTime = traversalTime;
     }
 
-    public int getTraversalTime() {
-        return traversalTime;
+    public int calculateTraversalTime() {
```

Comment:
@jfabi would you be so kind to review business logic here and in `calculateWheelchairTraversalTime` below?